venue/views.py

```diff
@@ -465,6 +471,11 @@ def confirm_email(request):
         user_profile = UserProfile.objects.get(user=token.user)
         user_profile.email_confirmed = True
         user_profile.save()
+        # Send email to Constant Contact
+        send_to_constant_contact(
```

Comment:
Also need to update email address if they change their email address.

Reply:
Addressed in my follow up commit. I'll invalidate those API key and access token. I'll create a new set and set it to you in private.

Reply:
Sounds good.

Reply:
Regarding updating when changing email address, I'm working on it now. I'll push shortly.

Reply:
This is addressed by my latest commit in this PR.